packages/material-ui-lab/src/Rating/Rating.js

```diff
@@ -90,7 +89,6 @@ export const styles = (theme) => ({
   },
   /* Styles applied to the icon wrapping elements. */
   icon: {
-    display: 'flex',
```

Comment:
It's for having the correct dimensions:

<img width="227" alt="Capture d’écran 2020-09-12 à 13 09 35" src="https://user-images.githubusercontent.com/3165635/92994269-3adfc900-f4f9-11ea-88a9-5542ab0f9cf8.png">

Reply:
How do you define "correct dimensions" here? I see that these are changed but as far as I can tell they're just arbitrary.

Reply:
"correct" meaning rectangular?

Reply:
Yes, rectangular to match the shape of the ratings: 

<img width="156" alt="Capture d’écran 2020-09-12 à 13 14 11" src="https://user-images.githubusercontent.com/3165635/92994364-dcffb100-f4f9-11ea-9edf-a83e332fbffc.png">

Reply:
Really need that visual regression test suite for states.

Reply:
On X side, we had the following plan to cover it:

- As an alternative to karma, use puppeteer to run tests.
- Use Puppeteer when Karma can't be used. Most often when there is a visual dimension or a high-level test API karma doesn't offer
- During puppeteer tests, take screenshots of visual states and send them with the other images to Argos-CI.

Maybe we could share the same solution here?

Reply:
It was just an off-hand remark. https://github.com/mui-org/material-ui/pull/21449 addresses this already.